### PR TITLE
move longrun to gpu

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Moves almost all of the longruns on clima GPU. The only jobs left on CPU are temporary jobs that are not very important. Also removes `longrun_aquaplanet_amip` as it's a bit redundant.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by szy21** [comment]
Testing it here: https://buildkite.com/clima/climaatmos-gpulongruns/builds/84

File: .buildkite/longruns_gpu/pipeline.yml [hunk 1]
```diff
@@ -136,7 +170,35 @@ steps:
...
+      - label: ":computer: amip target diagnostic edmf"
+        command:
+          - srun julia --project=examples examples/hybrid/driver.jl --config_file $CONFIG_PATH/$$JOB_NAME.yml
+        artifact_paths: "$$JOB_NAME/*"
+        agents:
+          slurm_gpus: 1
+          slurm_time: 12:00:00
+        env:
+          JOB_NAME: "amip_target_diagedmf"
+
```
Comment (Sbozzolo) [inline .buildkite/longruns_gpu/pipeline.yml:204]: Please, add newline here

**Comment by Sbozzolo** [comment]
At least one of the jobs contains a `Matrix` (that is not GPU compatible)

**Comment by szy21** [comment]
This job (with varying surface albedo) fails: https://buildkite.com/clima/climaatmos-gpulongruns/builds/84#018e2fc1-6d22-4e69-a918-7ecb3831f69d/150-231. @Sbozzolo Would you like to take a quick look to see if it's easy to fix? Otherwise I can move this back to CPU and open an issue.

File: .buildkite/longruns_gpu/pipeline.yml (hunk 1 above)
Comment (szy21) [reply .buildkite/longruns_gpu/pipeline.yml:204]: Ah, I thought I added a new line, but apparently I added a line with spaces. Thanks!

**Comment by szy21** [comment]
All other jobs are running, so I canceled them.

**Comment by szy21** [comment]
It's working: https://buildkite.com/clima/climaatmos-gpulongruns/builds/86#018e300a-ad52-4615-baa5-588cf4760937

**Pull request merged**